helper/Data.php

```diff
@@ -100,6 +108,43 @@ public function getOriginKeyForOrigin()
         return $originKey;
     }
 
+    /**
+     * @param $store
+     * @param $country
+     * @return array
+     */
+    public function fetchPaymentMethods($countryCode, $amount, $currency, $shopperReference, $shopperLocale)
+    {
+        $merchantAccount = \Configuration::get('ADYEN_MERCHANT_ACCOUNT');
+
+        if (!$merchantAccount) {
+            return [];
```

Comment:
would be nice to show or log a message that the merchant account is not configured therefore no payment methods are retrieved.